spec/myst/string_spec.mt

```diff
@@ -145,3 +145,98 @@ describe("String#to_s") do
     assert("\tone\ttwo".to_s == "\tone\ttwo")
   end
 end
+
+describe("String#chars") do
+  it("Returns a List with chars in a string") do
+    assert("abc".chars == ["a", "b", "c"])
+    assert("yay".chars == ["y", "a", "y"])
+  end
+end
+
+describe("String#downcase") do
+  it("Returns a lowercased version of itself") do
+    assert("HELLO".downcase == "hello")
+  end
```

Comment:
Adding some more tests for other cases would be nice. Maybe another test for a string that's mixed case, and another for a string that's already all downcased.

Same would go for `upcase`.

Reply:
I'd like to see these split into separate `it` blocks as well, so that test failures (at least, once the Spec library has been improved) can give accurate messages about exactly what failed. Same goes for the other specs with multiple `assert` expressions.

For example: here, I'd write the specs as:

```myst
it("returns a lowercased version of itself") do
  assert("HELLO".downcase == "hello")
end

it("downcases all characters when the original string is mixed case") do
  assert("hElLo".downcase == "hello")
end

it("returns the same string when all characters are already downcased") do
  assert("hello".downcase == "hello")
end

it("returns an empty string when the original string is empty") do
  assert("".downcase == "")
end

it("preserves white space in string") do
  assert("Bobby Fischer".downcase == "bobby fischer")
end

it("preserves special characters in the string") do
  assert("\n\t!^$#Q$#^&*<>\r\n".downcase == "\n\t!^$#Q$#^&*<>\r\n")
end
```